CodeGeneration/Sources/SyntaxSupport/Child.swift

```diff
@@ -147,8 +147,8 @@ public class Child {
 
   /// If a classification is passed, it specifies the color identifiers in
   /// that subtree should inherit for syntax coloring. Must be a member of
-  /// SyntaxClassification in SyntaxClassifier.h.gyb
-  /// If force_classification is also set to true, all child nodes (not only
+  /// ``SyntaxClassification`` in `SyntaxClassifier.h.gyb`
+  /// If `force_classification` is also set to true, all child nodes (not only
```

Comment:
I know that’s not really what you were after in this PR but since you’re already touching this, let’s update it so it’s not out-of-date

```suggestion
  /// ``SyntaxClassification``.
  /// If `forceClassification` is also set to true, all child nodes (not only
```